director/background_tasks.go

```diff
@@ -111,11 +111,10 @@ func pushAll() error {
 	var dbDevices []types.Device
 	now := time.Now()
 
-	DelaySeconds, HalfDelaySeconds := getDelay()
+	DelaySeconds, _ := getDelay()
```

Comment:
Why does `getDelay()` return two values? Can we return just one value and do `(getDelay() / 2)` in place? 

- I realize it's possible a lot of calls to getDelay might need to be changed. :/ 
- If you are willing do the above, can we make the function name more description? What is the context for this delay?